src/main/java/org/folio/service/budget/BudgetService.java

```diff
@@ -219,23 +218,14 @@ public void clearReadOnlyFields(Budget budgetFromNew) {
   }
 
   public Future<Void> checkBudgetHaveMoneyForTransaction(Transaction transaction, DBClient client) {
-    if (transaction.getFromFundId() == null) {
+    if (transaction.getFromFundId() == null || transaction.getTransactionType() == Transaction.TransactionType.TRANSFER) {
```

Comment:
To improve readability please replace with java.util.Objects.isNull static method this statement : "transaction.getFromFundId() == null"

Reply:
Ok, I will fix, thanks